### PR TITLE
Feature/#5

### LINKED ISSUE #5: implement user model

### PR DESCRIPTION
**Pull request by SLDem**

### CODE REVIEW

File: user/views.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import render
+from django.views.generic import DetailView, FormView, DeleteView, ListView
+from django.http import JsonResponse
+
+from .models import User
+from lists.models import ToDoList
```
Comment (StepanTchynetskyi) [inline user/views.py]: from list_project.models import List

File: user/views.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+class ProfileView(DetailView):
+    model = User
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        user = self.get_object()
+        lists = ToDoList.objects.filter(user_id=user.pk)
```
Comment (StepanTchynetskyi) [inline user/views.py]: ToDoList change to List

**Review by StepanTchynetskyi — CHANGES_REQUESTED** [review]

**Review by StepanTchynetskyi — APPROVED** [review]

File: user/models.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        )
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_user(self, email, password, **extra_fields):
+        return self._create_user(email, password, **extra_fields)
+
+
+class User(AbstractBaseUser):
```
Comment (lhalam) [inline user/models.py]: rename to CustomUser

File: user/tests.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (lhalam) [inline user/tests.py]: delete this file

**Review by lhalam — APPROVED** [review]

**Pull request merged**